tests/roles/backend_services/tasks/main.yaml

```diff
@@ -37,6 +37,10 @@
     {% if placement_password %}
         oc set data secret/osp-secret "PlacementPassword={{ placement_password }}"
     {% endif %}
+    {% if heat_password %}
+        oc set data secret/osp-secret "HeatPassword={{ heat_password }}"
+        oc set data secret/osp=secret "HeatAuthEncryptionKey={{ heat_auth_encryption_key }}"
```

Comment:
there is a typo here, `oc set data secret/osp=secret "HeatAuthEncryptionKey={{ heat_auth_encryption_key }}"` should be `oc set data secret/osp-secret "HeatAuthEncryptionKey={{ heat_auth_encryption_key }}"`

Reply:
Ah, fixed. Thanks for pointing that out.